cpp/dolfinx/mesh/Mesh.cpp

```diff
@@ -79,15 +79,20 @@ Mesh mesh::create(MPI_Comm comm,
                                cells);
 
   // Compute the destination rank for cells on this process via graph
-  // partitioning
+  // partitioning. Always get the ghost cells via facet, though these may be
```

Comment:
What if the user requests ghosted by vertex?

Reply:
It's currently not supported. I will add some kind of message or error.